buildSrc/win.gradle

```diff
@@ -140,67 +138,48 @@ if (sourceDateEpoch != null) {
 }
 
 // Remove C++ static linking if not on VS2010
-if (WINDOWS_VS_VER != "100") {
-    ccFlags -= ["/D_STATIC_CPPLIB", "/D_DISABLE_DEPRECATE_STATIC_CPPLIB"]
-}
+ccFlags -= ["/D_STATIC_CPPLIB", "/D_DISABLE_DEPRECATE_STATIC_CPPLIB"]
```

Comment:
The "if not on VS2010" part of the comment is obsolete.

Reply:
I'll remove it.